helpdesk-bot/src/test/java/org/symphonyoss/symphony/bots/helpdesk/bot/it/steps/TicketSteps.java

```diff
@@ -360,16 +372,24 @@ public void closeOtherTicket(String username)
     Thread.sleep(5000L);
   }
 
-  @Then("bot can verify there are no agents in the ticket room")
-  public void emptyRoom() throws SymException {
+  @Then("bot can verify there are $condition agent in the ticket room")
```

Comment:
You should create two methods to verify the room members:

- verifyOnlyBotInTicketRoom
- verifyAgentInTicketRoom

The second one you should provide the agent user as a parameter.